tesseract/tesseract_collision/CMakeLists.txt

```diff
@@ -114,6 +114,8 @@ install(DIRECTORY include/${PROJECT_NAME}
 
 install(FILES package.xml DESTINATION share/${PROJECT_NAME})
 
+file(WRITE ${CMAKE_INSTALL_PREFIX}/share/ament_index/resource_index/packages/${PROJECT_NAME} "")
```

Comment:
I recommend adding a comment explaining why this is required similar to what you wrote in the description on the PR.

Reply:
I added an explanatory comment with some info about the workaround and the other steps required to find the plugins from ROS2.